package.json

```
@@ -78,6 +78,7 @@
     "babel-loader": "^8.2.2",
     "babel-plugin-styled-components": "2.0.7",
     "babel-plugin-unassert": "^3.0.1",
+    "clsx": "^2.1.1",
```

```suggestion
    "clsx": "2.1.1",
```
Just for sanity's sake if that's ok with you 😃